kubespawner/spawner.py

```diff
@@ -472,7 +511,7 @@ def poll(self):
 
     @gen.coroutine
     def start(self):
-        if self.storage_class:
+        if self.user_storage_class && self.user_storage_capacity:
```

Comment:
yuvi, if we use `or` instead of `and`, like `self.user_storage_class is not None or self.user_storage_capacity is not None`, the pvc will be created if either the storage class or storage capacity is set. But if one of them is set, and the other is not, then the pvc will still try to be created. Is this what we want?